lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -20,18 +20,21 @@
     % elif not credit_status["purchased"]:
         <p class="credit-eligibility-msg">
             % if credit_status["deadline"] < datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=30):
-                ${_("The opportunity to purchase credit for this course expires on {deadline}. You've worked hard - don't miss out!").format(deadline=get_default_time_display(credit_status["deadline"]))}
+                ${_("Your eligibility to receive {provider_name} credit is expiring on {deadline}. Don't miss out!").format(
+                    provider_name=credit_status["provider_name"],
+                    deadline=get_default_time_display(credit_status["deadline"]),
+                )}
             % else:
-                ${_("Congratulations - you have met the requirements for credit in this course!")}
+                ${_("You are now eligible for {provider_name} Credit. Congratulations!").format(provider_name=credit_status["provider_name"])}
             % endif
         </p>
         <div class="purchase_credit">
             ## TODO: set the URL for the link to the provider selection page on the E-Commerce service
-            <a class="btn purchase-credit-btn" href="#" target="_blank" data-course-key="${credit_status["course_key"]}">${_("Purchase Course Credit")}</a>
+            <a class="btn credit-btn purchase-credit-btn" href="#" target="_blank" data-course-key="${credit_status["course_key"]}">${_("Get credit")}</a>
         </div>
     % elif credit_status["request_status"] in [None, "pending"]:
```

Comment:
Based on Andy's comment in the JIRA ticket, I think we need to separate out these states.

1) `credit_status["request_status"] is None`: means that the user has paid for credit, but has not yet requested credit from a provider.

2) `credit_status["request_status"] == "pending": means that the user has paid for credit AND initiated a request from the provider.

We already have messaging for (2); in this PR, please add messaging for (1).  In Andy's comment, he suggests adding text like: "You still need to visit <partner name> to complete credit".

He also suggests changing the "Purchase" button to "Complete Credit".  The "Complete Credit" button is somewhat complex, because it needs to (a) call an API end-point (which isn't yet exposed) to initiate a credit request, (b) in JavaScript, dynamically construct a form based on the returned parameters from the end-point, and (c) automatically submit the form.  The API end-point you will need to expose is mostly implemented, but needs a URL and Django view: https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/credit/api/provider.py#L55

Please let me know if I can clarify any of these details.